src/App.js

```diff
@@ -1,26 +1,126 @@
 import React from 'react';
-import logo from './logo.svg';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import Search from './components/Search';
+
+import foods from './foods.json';
+
+import 'bulma/css/bulma.css';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    cloneFoods: foods,
+    addForm: false,
+    todaysFoods: [],
+    totalCal: 0,
+  };
+
+  // Form handler //
+  handleAdd = (e) => {
+    e.preventDefault();
+    let name = e.target.addFoodName.value;
+    let image = e.target.addFoodImage.value;
+    let calories = e.target.addFoodCalories.value;
+
+    let newFood = {
+      name: name,
+      image: image,
+      calories: calories,
+      quantity: 0,
+    };
+    this.setState({
+      cloneFoods: [...foods, newFood],
```

Comment:
**foods** is the original object, so when you try to add a new element you will never see the previous ones added. So instead of **...foods** try using the current state: **...this.state.cloneFoods**